_posts/2020-01-22-cockpit-211.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 211
+author: mvo
+date: 2020-01-22
+tags: cockpit linux
+slug: cockpit-211
+category: release
+summary: Cockpit with certificate improvements and more
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 211.
+
+### Better support for various certificates formats
```

Comment:
For the tag I'm using "various TLS certificate formats" (note the singular)

Reply:
Fixed.